receiver/jmxreceiver/config.go

```diff
@@ -11,55 +11,81 @@ import (
 	"io"
 	"os"
 	"sort"
+	"strconv"
 	"strings"
 	"time"
 
 	"go.opentelemetry.io/collector/config/configopaque"
 	"go.opentelemetry.io/collector/exporter/exporterhelper"
+	"go.uber.org/multierr"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
+// jmxGathererMainClass the class containing the main function for the JMX Metric Gatherer JAR
+var jmxGathererMainClass = "io.opentelemetry.contrib.jmxmetrics.JmxMetrics"
+
+// jmxScraperMainClass the class containing the main function for the JMX Scraper JAR
+var jmxScraperMainClass = "io.opentelemetry.contrib.jmxscraper.JmxScraper"
+
 type Config struct {
-	// The path for the JMX Metric Gatherer uber JAR (/opt/opentelemetry-java-contrib-jmx-metrics.jar by default).
+	// The path for the JMX Metric Gatherer or JMX Scraper JAR (/opt/opentelemetry-java-contrib-jmx-metrics.jar by default).
```

Comment:
There are a few new configuration options that are only supported by jmx-scraper and need to be added here ([doc](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-scraper/README.md)) :
- `otel.jmx.target.source` which must be set to `legacy` by default to make jmx-scraper use metric definitions close to current jmx-gatherer metrics.
- `otel.jmx.config` that is a comma-semarated list of yaml files to use for custom metrics definitions (this is a new feature and does not introduce any security risk like gatherer with groovy executable code).

Reply:
Configuration options updated :+1: